packages/badge/Badge.test.tsx

```diff
@@ -1,18 +1,51 @@
-import { Badge } from './Badge';
-
+import { Badge } from './index';
 import { h, mount } from 'bore';
-import { emit } from 'skatejs';
 
 describe( Badge.is, () => {
 
   it( 'should render', () => {
 
-    console.warn('Missing tests for Badge!');
-
     return mount(
       <Badge/>
-    ).wait();
+    ).wait((element) => element.has('c-badge') );
 
   });
 
+  it( 'should render rounded', () => {
```

Comment:
Captain here, this pattern is taken from bore readme. Even that martin is right and test should usually have at least one assertion in this case it is valid way of testing asynchronous code. I don't see any advatages of adding is.equal(true) everywhere.